### DISCUSSION TITLE
Hidden cursor mod

### DISCUSSION THREAD

**Discussion by MightyHelper** [ppy/osu#13082] [Ideas] [answered]
**Describe the new feature:** 
Mod that hides the cursor.
**Proposal designs of the feature:** 
Things to take into account
- Bad name(Pls suggest)
- RIP Mouse users (So maybe unranked?)
- Score multiplier (Could be unranked soo?)
- Maybe show light cursor trail?
- Before the song starts need cursor to adjust dim and so on...
- Pause every few secs to see the cursor on pause screen

**Comment by smoogipoo** [comment]
Could be implemented as a setting for the existing hidden mod.

**Comment by Semro** [comment]
I would like to give it a try to implement this mod.

Starting with a completely hidden cursor seems very challenging, maybe it's better smoothly fade out the cursor while the combo is rising and show it on combo breaks (similar to "Muted" mod). Show cursor during breaks and spinners.
Since we are playing around cursor transparency I have no better idea than to call it "Ghost" mod.

Possible settings:
- Fading out duration
- Final cursor transparency

**Reply by frenzibyte** [reply]
As mentioned above, it can be a setting in the existing hidden mod.

**Reply by Semro** [reply]
By setting it means to be as a `MultiMod` with Hidden?

**Reply by bdach** [reply]
That's not the same thing. See `SettingSourceAttribute` and its usages.

**Reply by Semro** [reply]
Maybe it looks like those mods are relevant to each other due to their name but I more see it as a separate mod with their own settings in the fun section.

**Comment by jaswon** [comment, marked as answer]
Hi! 👋 I recently worked on this, and the initial pass has been merged into master as the "No Scope" mod https://github.com/ppy/osu/pull/15111
